src/components/about/About.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import styled from 'styled-components'
+import logo from '../../assets/logo.png'
+
+const Logo = styled.img`
+    display: block;
+    margin: auto;
+`
+const Intro = styled.p`
+    text-align: center;
+`
+const Section = styled.div`
+    column-count: 2;
+`
+
+const About = ({ intro }) => {
```

Comment:
Filename could be more specific to the template

Reply:
Do you have any suggestions for this?

Reply:
Maybe `DuoColumnWithTopAbout`? That might be too long